app/models/application.rb

```diff
@@ -0,0 +1,27 @@
+class Application < ApplicationRecord
+  has_many :pet_applications
+  has_many :pets, through: :pet_applications
+  belongs_to :admin
+  validates :name, presence: true
+  validates :address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zipcode, presence: true
+  validates :description, presence: true
+
+  def add_pet(pet_id)
+    pet = Pet.find(pet_id)
+    pets << pet
```

Comment:
Nice encapsulation of this logic! This is not strictly necessary from an MVC perspective but I do think it cleans up this functionality into a nice modular action